sechub-commons-model/src/test/java/com/mercedesbenz/sechub/commons/model/SecHubConfigurationModelValidatorTest.java

```diff
@@ -1316,6 +1316,56 @@ void model_has_no_header_values_specified_has_error(String missingHeaderValue) {
         assertHasError(result, SecHubConfigurationModelValidationError.WEB_SCAN_NO_HEADER_VALUE_DEFINED);
     }
 
+    @ParameterizedTest
+    @NullSource
+    @EmptySource
+    void model_has_header_value_only_from_file_ref_specified_has_no_error(String explicitHeaderValue) {
+        /* prepare */
+        List<HTTPHeaderConfiguration> httpHeaders = createListWithOneHeaderAndOneOnlyForUrl("Authorization", explicitHeaderValue, "https://mywebapp.com");
+        // add header file ref
+        httpHeaders.get(0).getNamesOfUsedDataConfigurationObjects().add("header-file-ref");
+
+        SecHubWebScanConfiguration webScan = new SecHubWebScanConfiguration();
+        webScan.headers = Optional.ofNullable(httpHeaders);
+        webScan.url = URI.create("https://mywebapp.com");
```

Comment:
Please use always `example.org` or `example.com`
```suggestion
        webScan.url = URI.create("https://example.com");
```

This should be done in every test here.